src/Meshes/buffer.ts

```diff
@@ -325,6 +325,16 @@ export class VertexBuffer {
         this._instanceDivisor = instanced ? divisor : 0;
     }
 
+    /*
+        Return a clone of this VertexBuffer that references the same data buffer.
+        Data buffers are reference counted so it will only be released once even if we have two VertexBuffers referencing it.
+    */
+    public clone(): VertexBuffer {
+        var useBytes = false; // No idea what to set here

```

Comment:
you can simply call the constructor with false as all the parameter are already set up